Source/WebCore/workers/service/ServiceWorkerRegistration.h

```diff
@@ -116,6 +122,8 @@ class ServiceWorkerRegistration final : public RefCounted<ServiceWorkerRegistrat
     void refEventTarget() final { ref(); }
     void derefEventTarget() final { deref(); }
 
+    Ref<ServiceWorkerContainer> protectedContainer() const;
```

Comment:
This should be `public`, right next to `container()`. Why make one public and not the other?